sdk/src/test/scala/com/horizen/account/receipt/ReceiptFixture.scala

```diff
@@ -35,19 +34,12 @@ trait ReceiptFixture {
       new EvmLog(address, topics, data)
     }
 
-  def createTestEthereumReceipt(txType: Integer, num_logs: Integer = 2, contractAddressPresence : Boolean = true, txHash: Array[Byte] = null, address: Array[Byte] = null): EthereumReceipt = {
-    var txHashTemp: Array[Byte] = new Array[Byte](32)
-
-    if(txHash != null) {
-      txHashTemp = txHash
-    }
-    else {
-      Random.nextBytes(txHashTemp)
-    }
+  def createTestEthereumReceipt(txType: Integer, num_logs: Integer = 2, contractAddressPresence : Boolean = true, txHash: Option[Array[Byte]], address: Array[Byte] = Address.addressZero().toBytes): EthereumReceipt = {
```

Comment:
You could add a default value for the `txHash` here, like for most of the other arguments, saving you from adding `txHash = None` everywhere.